### ISSUE TITLE
Set Preheater duration - working - no issue

### ISSUE THREAD

**Issue by msandvold** [skodaconnect/homeassistant-skodaconnect#188]
I've had some problems getting the "Pre heater duration" to work.

**First step i OK**:

In configuration.yaml:

input_number:
  pheater_duration:
    name: "Pre-heater duration"
    initial: 20
    min: 10
    max: 60
    step: 10
    unit_of_measurement: min

Create the automation, in yaml or via GUI editor. You can find the device id by going to Settings->Devices & Services and then clicking on the device for the Skodaconnect vehicle. The device ID will show in the web browser address bar after "/config/devices/device/". The ID can also be found by using the GUI services editor under developer tools. Choose one of the skodaconnect services and choose your vehicle. Change to YAML mode and copy the device ID.

**this is my automation in yaml**

It was the duration - state from input_number.pheater_duration that resulted in a number with one decimal -> 10.0 / 20.0 / 30.0. So I had to edit the duration to get the expxted state correct.

New automation -> change to yaml mode ->**paste code** -> replace "Your Device ID for vehicle" with Your ID, as described above.

```
alias: Pre-heater duration
description: ""
trigger:
  - platform: state
    entity_id: input_number.pheater_duration
condition: []
action:
  - service: skodaconnect.set_pheater_duration
    data_template:**
      device_id: <Your Device ID for vehicle>
      duration: "{{ (states('input_number.pheater_duration') | float ) | round(0) }}"
mode: single
```

**You need to set duration before BEFORE starting the heater/ventilation.**

**Comment by NickSutton** [comment]
I get an error when calling this service: 

````
This error originated from a custom integration.

Logger: homeassistant.helpers.script.websocket_api_script
Source: custom_components/skodaconnect/__init__.py:282 
Integration: Skoda Connect (documentation, issues) 
First occurred: 14:10:52 (1 occurrences) 
Last logged: 14:10:52

websocket_api script: Error executing script. Unexpected error for call_service at pos 1: Could not find associated coordinator for given vehicle
Traceback (most recent call last):
  File "/config/custom_components/skodaconnect/__init__.py", line 280, in get_car
    dev_coordinator = hass.data[DOMAIN][conf_entry]['data'].coordinator
                      ~~~~~~~~~~~~~~~~~^^^^^^^^^^^^
KeyError: '1379863f3a9e6a1bb696ebbbf92c5e70'

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "/usr/src/homeassistant/homeassistant/helpers/script.py", line 468, in _async_step
    await getattr(self, handler)()
  File "/usr/src/homeassistant/homeassistant/helpers/script.py", line 703, in _async_call_service_step
    response_data = await self._async_run_long_action(
                    ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/usr/src/homeassistant/homeassistant/helpers/script.py", line 665, in _async_run_long_action
    return long_task.result()
           ^^^^^^^^^^^^^^^^^^
  File "/usr/src/homeassistant/homeassistant/core.py", line 1974, in async_call
    response_data = await coro
                    ^^^^^^^^^^
  File "/usr/src/homeassistant/homeassistant/core.py", line 2011, in _execute_service
    return await target(service_call)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/config/custom_components/skodaconnect/__init__.py", line 412, in set_pheater_duration
    car = await get_car(service_call)
          ^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/config/custom_components/skodaconnect/__init__.py", line 282, in get_car
    raise SkodaConfigException('Could not find associated coordinator for given vehicle')
skodaconnect.exceptions.SkodaConfigException: Could not find associated coordinator for given vehicle

````

**Comment by dvx76** [comment]
@NickSutton that error seems unrelated to the issue reported here. Please create a new issue. Include if the rest of the integration is working correctly... it looks like there's a problem with the stored configuration of the integration.
